### DISCUSSION TITLE
What is the current best way to handle file dialogues?

### DISCUSSION THREAD

**Discussion by ThomasAlban** [bevyengine/bevy#8829] [Q&A] [answered]
How should I approach file dialogues in bevy? I'm using bevy_egui and want to be able to click a button which will open a file dialogue and store the file path. I have read #2032, and someone suggested some code using rfd, but this is out of date. I'm wondering if there is a simpler way of doing this?

**Comment by dimvoly** [comment, marked as answer]
I've had luck cloning [egui_file](https://github.com/Barugon/egui_file) and then customizing it for my needs.

All you get out of it is a path, which is all you really need.

**Reply by ThomasAlban** [reply]
What exactly did you need to change in egui_file to make it work?

**Reply by dimvoly** [reply]
From memory, there was some error type that didn't have the right traits on it, so I had to change that to make it compatible with bevy. Once you import `egui_file`, the compiler should highlight what's wrong, and it's not too difficult from there.
